src/components/shared/Draggable.js

```diff
@@ -116,7 +121,21 @@ export class Draggable extends React.PureComponent<Props, State> {
 
   render() {
     const props = Object.assign({}, this.props);
-    if (this.state.dragging) {
+    if (
+      props.className.includes('RangeStart') &&
+      'draggingStart' in props.value
```

Comment:
This does not belong in the generic Draggable component. Instead, you could handle this in the CSS - just set draggingStart / draggingEnd classes on the parent.

Reply:
I set the draggingStart / draggingEnd classes on the parent as requested.

In the CSS, the `:not(.draggingStart, .draggingEnd) > .timelineSelectionGrippyRangeEnd.dragging` selector is to handle the edge case where the mouse is pressed on the RangeEnd grippy, then the pointer is moved down (above the first track) without moving it horizontally. In that case, because `selectionDeltas.endDelta` is 0, the `draggingEnd` class doesn't get set, so when the grippy is no longer hovered, it becomes light grey, until it is moved horizontally.